prime-router/src/main/resources/db/migration/R__digest_index.sql

```diff
@@ -9,6 +9,6 @@
  */
 
 CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_report_file_blob_digest
-  on report_file
-  using btree(blob_digest)
-;
\ No newline at end of file
+    on report_file
+        using btree(blob_digest)
+;
```

Comment:
If you use `R__` instead of `Vxx__` as the prefix to your flyway script it will always rerun your script for you on each migration, which is handy to know in the future. It of course means your script *MUST* be idempotent. This was another suggestion to try. So was adding `NT` to the front, which also __DID NOT__ work.